internal/module/services/version_manager/version_manager.go

```diff
@@ -194,6 +199,38 @@ func (s *VersionManagerService) PublishVersion(ctx context.Context, request *ser
 		return nil, PublishModuleVersionError
 	}
 
+	// PUBLISH RELEASE
+	parsedVersion, err := versions.ParseVersion(strings.ReplaceAll(request.Module.GetVersion(), "v", ""))
+	if err != nil {
+		span.RecordError(err)
+		return nil, err
+	}
+
+	if parsedVersion.GreaterThan(DevelopmentVersion) {
+		moduleAddress := strings.Split(request.Module.GetName(), "/")
+		orgName := moduleAddress[0]
+
+		connVersion, err := services.CreateGRPCConnection(s.ReleaseServiceEndpoint)
+		if err != nil {
+			span.RecordError(err)
+			log.Printf("Failed to connect to '%s': %v", s.ReleaseServiceEndpoint, err)
+		}
+		defer closeClient(connVersion)
+
+		client := releaseSvc.NewPublisherClient(connVersion)
+		_, err1 := client.Publish(ctx, &releasePkg.PublishRequest{
+			Name:         request.Module.GetName(),
+			Version:      request.Module.GetVersion(),
+			Type:         *aws.String("module"),
+			Organization: orgName,
+		})
+
+		if err1 != nil {
+			span.RecordError(err)
+			return nil, PublishReleaseVersionError
```

Comment:
Failing to publish a release isn't a fatal error as we've already created everything else (ie the module will appear in Terrarium).  We could register a span event to flag that no release was published but we don't need to signal an error here.